ci/prow/config.yaml

```diff
@@ -242,10 +242,8 @@ presubmits:
         command:
         - "/coverage"
         args:
-        - "--postsubmit-gcs-bucket=knative-prow"
         - "--postsubmit-job-name=post-knative-serving-go-coverage"
         - "--artifacts=$(ARTIFACTS)"
-        - "--cov-threshold-percentage=80"
```

Comment:
My fault, I misunderstood.
I prefer not to make this change though, since `cov-threshold-percentage` is written in the template, it would make the code less cleaner by making it optional. 
I did change the default value to `50` and used `80` to overwrite it in `config_knative.yaml`, I think it looks more natural.